mmv1/products/compute/terraform.yaml

```diff
@@ -2845,6 +2845,37 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_value: :TLS_1_0
       warnings: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
+  # Find that Region SSL Policy needs the same custom code as SSL Policy
+  RegionSslPolicy: !ruby/object:Overrides::Terraform::ResourceOverride
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants:  'templates/terraform/constants/region_ssl_policy.erb'
+      resource_definition: 'templates/terraform/resource_definition/region_ssl_policy.erb'
+      #update_encoder: 'templates/terraform/update_encoder/region_ssl_policy.erb'
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      customFeatures: !ruby/object:Overrides::Terraform::PropertyOverride
+        is_set: true
```

Comment:
Just curious if this would work as a list? Lists are typically easier to maintain if we can get away with it, but if its unordered, we'll have to keep it as a set (same with `enabled_features`)

Reply:
Not 100% sure... but the features are mapped to cipher suites. So would assume the intersection (sets!) of provided values and valid cipher suites would need to found.